docs/edit-profile.md

```diff
@@ -2,7 +2,7 @@
 
 TechPass users can edit their TechPass profile anytime by logging into the TechPass portal.
```

Comment:
"anytime" necessary?

Is it "TechPass Portal" or "TechPass portal"?

Reply:
fixed.